### PR TITLE
Updates for v23.12 release

### PR DESCRIPTION
**Pull request by raydouglass**
Updates for v23.12 release

Also adds back the announcements for release & Pascal deprecation.

Depends on rapidsai/docs#461

### CODE REVIEW

**Comment by raydouglass** [comment]
Should the `pip install` commands specify the version like the conda commands do?

File: content/_index.html [hunk 1]
```diff
@@ -33,10 +33,10 @@
 
   <!-- Announcements Shortcode (0-3x)-->
   <!-- UPDATE HERE -->
-  <!-- {{< announcements
-    text-2="RAPIDS 23.10 Released" link-2="https://docs.rapids.ai/install#selector"
-    text-3="Notice: Depreciation for Pascal GPUs" link-3="https://docs.rapids.ai/notices"
-  >}} -->
+  {{< announcements
+    text-2="RAPIDS 23.12 Released" link-2="https://docs.rapids.ai/install#selector"
+    text-3="Notice: Depreciation for Pascal GPUs" link-3="https://docs.rapids.ai/notices/rsn0034/"
```
Comment (bdice) [inline content/_index.html:38]:
```suggestion
    text-3="Notice: Deprecation for Pascal GPUs" link-3="https://docs.rapids.ai/notices/rsn0034/"
```

**Comment by bdice** [comment]
> Should the `pip install` commands specify the version like the conda commands do?

Currently it says
```
pip install cudf-cu11 dask-cudf-cu11 --extra-index-url=https://pypi.nvidia.com
  pip install cuml-cu11 --extra-index-url=https://pypi.nvidia.com
  pip install cugraph-cu11 --extra-index-url=https://pypi.nvidia.com
```

I would prefer to:
- Specify the index URL only once (use one install command)
- Use cu12 instead of cu11 by default
- Add version pinnings like `==23.12.*`.

```
pip install cudf-cu12==23.12.* dask-cudf-cu12==23.12.* cuml-cu12==23.12.* cugraph-cu12==23.12.* --extra-index-url=https://pypi.nvidia.com
```

I think this looks fine when rendered, too:
image

**Comment by bdice** [comment]
Or alternatively we can make the "short pip install" look more like the install selector:
```
pip install \
  --extra-index-url=https://pypi.nvidia.com \
  cudf-cu12==23.12.* \
  dask-cudf-cu12==23.12.* \
  cuml-cu12==23.12.* \
  cugraph-cu12==23.12.*
```

**Comment by raydouglass** [comment]
> Or alternatively we can make the "short pip install" look more like the install selector:

I like this. I've pushed a commit to add that.

File: content/_index.html [hunk 2]
```diff
@@ -138,13 +138,16 @@ <h3 class="mt-6 text-white">Install with Conda</h3>
   bash Miniconda3-latest-Linux-x86_64.sh</code></pre>
               <br>
               <p><strong>2.</strong> Then quick install RAPIDS with:</p>
-              <pre class="highlight use-white-copy"><code>conda create --solver=libmamba -n rapids-23.10 -c rapidsai -c conda-forge -c nvidia rapids=23.10 python=3.10 cudatoolkit=11.8</code></pre>
+              <pre class="highlight use-white-copy"><code>conda create --solver=libmamba -n rapids-23.12 -c rapidsai -c conda-forge -c nvidia rapids=23.12 python=3.10 cudatoolkit=11.8</code></pre>
 
               <h3 class="mt-6 text-white">Install with pip</h3>
               <p><strong>1.</strong> Install via pip channels:</p>
```
Comment (bdice) [inline content/_index.html:144]: This is a single-step installation. We can remove the **1.**
```suggestion
              <p>Install via pip channels:</p>
```

**Review by bdice — APPROVED** [review]
A couple small fixes. Otherwise LGTM.

**Pull request merged**